pyproject.toml

```diff
@@ -21,7 +21,7 @@ dependencies = [
   "appdirs",
   "av",
   "cloudpathlib[s3]",
-  "botocore>=1.34.0",
+  "botocore",
```

Comment:
Could we just remove this now, since it's an indirect dependency?

Reply:
good call